lib/resources/r4/appointment.yaml

```diff
@@ -14,37 +14,86 @@ fields:
   note: resourceType must be Appointment
   url: http://hl7.org/fhir/r4/json.html#resources
 
-- name: slot
+- name: status
   required: 'Yes'
-  type: Reference (Slot)
-  description: The availability to which the appointment is booked.
+  type: code
+  description: A code representing the status of the new Appointment.
   action: create
   example: |
     {
-      "slot": [
+      "status": "booked"
+    }
+  note: When creating a new appointment, the status must be `booked` or `proposed`.
+
+- name: serviceType
+  required: 'No'
+  type: CodeableConcept
+  description: The specific service that is to be performed during this appointment.
+  action:
+  - read
```

Comment:
read? there shouldn't be a read table...

Reply:
https://github.com/cerner/fhir.cerner.com/pull/392/commits/741818815c57461d0bc7048e1446e6114838fedc